tdvt/tdvt/version.py

```diff
@@ -1,2 +1,3 @@
-__version__ = '2.6.8'
+__version__ = '2.6.9b1'
```

Comment:
```suggestion
__version__ = '2.6.9'
```Looks like this will go into artifactory as 2.6.9